kit/subgraph/src/capped/fetch/capped.ts

```
@@ -0,0 +1,19 @@
+import { Address, BigInt } from "@graphprotocol/graph-ts";
```

Biome (1.9.4)

[error] Do not shadow the global "BigInt" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)